### DISCUSSION TITLE
umi+react，运行之后报错，我感觉是把我所有的请求都扼杀了

### DISCUSSION THREAD

**Discussion by Lier1210** [umijs/umi#11366] [Q&A]
大家看图吧，我不知道怎么描述了，找了很久找不到答案了
![image](https://github.com/umijs/umi/assets/112168368/4b5d4446-c582-4157-a310-5990aeea7d29)

**Comment by Lier1210** [comment]
真的需要，sos！！！

**Reply by fz6m** [reply]
排查一下自己的请求服务和请求方法是否有问题导致响应取不到。

如有问题，需要提供一个最小复现。
